### PR TITLE
QAE-328 - E2E test case for footer builder background image

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for footer builder background image

### Screenshots

https://share.getcloudapp.com/DOumgbwo

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/footer-builder/background-image.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'footer builder backgeround image setting in customizer', () => {
+	it( 'background image for desktop should apply correctly', async () => {
+		const footerBuilder = {
+			'section-footer-builder-layout-padding': 60,
+			'footer-bg-obj-responsive': {
+				desktop: {
+					'background-image': 'https://images.ctfassets.net/hrltx12pl8hq/4CH9hYoMSORDuL8BXgX0Br/c2520603e3d2616468161c38519c9446/shutterstock_631575830.jpg?fit=fill&w=2560&h=400&fm=webp',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/background-image.test.js:11]: Check to add different images for responsive images.

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js:11]: Added different images

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'Footer builder backgeround image setting in customizer', () => {
+	it( 'background image for desktop should apply correctly', async () => {
+		const FooterBuilder = {
+			'section-footer-builder-layout-padding': 60,
+			'footer-bg-obj-responsive': {
+				desktop: {
+					'background-image': 'https://images.ctfassets.net/hrltx12pl8hq/4CH9hYoMSORDuL8BXgX0Br/c2520603e3d2616468161c38519c9446/shutterstock_631575830.jpg?fit=fill&w=2560&h=400&fm=webp',
```
Comment (premanshup) [inline tests/e2e/specs/customizer/footer-builder/background-image.test.js:11]: @SimranChougule1 Please use all the images from recommended sources - https://make.wordpress.org/themes/handbook/review/resources/#recommended-websites-for-images. @SimranChougule1 Can you replace all the images added with the websites mentioned here?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js]: added resources suggested by you but image is not visible on front end, @GauravKhupse is checking this problem

Comment (GauravKhupse) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js]: This issue in our Astra setting, I have fixed branch:- **footer-global-color-issue**, I am adding task in Astra.

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'Footer builder backgeround image setting in customizer', () => {
+	it( 'background image should apply correctly', async () => {
+		const FooterBuilderBgImage = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/background-image.test.js:7]:
```suggestion
		const footerBuilderBgImage = {
```

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js]: Done chnages

**Review by GauravKhupse — APPROVED** [review]

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'Footer builder backgeround image setting in customizer', () => {
```
Comment (premanshup) [inline tests/e2e/specs/customizer/footer-builder/background-image.test.js:5]:
```suggestion
describe( 'Footer builder background image setting in customizer', () => {
```

**Review by premanshup — APPROVED** [review]

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js (hunk 4 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js]: Resolved

File: tests/e2e/specs/customizer/footer-builder/background-image.test.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'Footer builder background image setting in customizer', () => {
+	it( 'background image should apply correctly', async () => {
+		const footerBuilderBgImage = {
+			'section-footer-builder-layout-padding': 60,
+			'footer-bg-obj-responsive': {
+				desktop: {
+					'background-image': 'https://pickupimage.com/free-photos/Combine-harvester/2358359',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/background-image.test.js:11]: desktop URL is not the proper URL, please update.

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/footer-builder/background-image.test.js]: resolved

**Pull request merged**